packages/medusa/src/api/routes/store/orders/get-order.js

```diff
@@ -0,0 +1,13 @@
+export default async (req, res) => {
+  const { id } = req.params
+
+  const orderService = req.scope.resolve("orderService")
+  const order = await orderService.retrieve(id)
+
+  if (!order) {
+    res.sendStatus(404)
+    return
+  }
```

Comment:
This isn't necessary - `retrieve` throws a `MedusaError.Types.NOT_FOUND` if the order isn't found and our error boundary converts this to a `404`